cmd/ctr/commands/content/content.go

```diff
@@ -539,14 +539,19 @@ func edit(context *cli.Context, rd io.Reader) (io.ReadCloser, error) {
 		tmp.Close()
 		return nil, err
 	}
-
-	if _, err := tmp.Seek(0, io.SeekStart); err != nil {
-		tmp.Close()
+	// need reopen tmp cause it is edited in another process
```

Comment:
the comment should be like:

The editor might recreate new file and override the original one. We should reopen the file.